products/veeam-backupandreplication

```diff
@@ -0,0 +1,111 @@
+---
+title: Veeam Backup & Replication
+category: app
+tags: veeam
+iconSlug: veeam
+permalink: /veeam-backupandreplication
+alternate_urls:
+-   /veeambackupandreplication
```

Comment:
```suggestion
alternate_urls:
-   /veeam-backup
```

Not sure `veeambackupandreplication` will be useful to anybody. Maybe `/veeam-backup` if this name is used as a contraction for the product ?